selfdrive/car/honda/carstate.py

```diff
@@ -326,7 +334,7 @@ def update(self, cp, cp_cam):
     # ******************* parse out can *******************
 
 
-    if self.CP.carFingerprint in (CAR.ACCORD, CAR.ACCORD_15, CAR.ACCORDH, CAR.INSIGHT): # TODO: find wheels moving bit in dbc
+    if self.CP.carFingerprint in (CAR.ACCORD, CAR.ACCORD_15, CAR.ACCORDH, CAR.CIVIC_BOSCH, CAR.CIVIC_BOSCH_DIESEL, CAR.CRV_HYBRID, CAR.INSIGHT): # TODO: find wheels moving bit in dbc
```

Comment:
I think that this might cause problems for the CIVIC_BOSCH (and others?), due to it not having the LEAD_DISTANCE in its DBC.  I haven't verified that, but I think that occurred in an OP branch of mine.